straxen/plugins/event_processing.py

```diff
@@ -195,7 +195,10 @@ def _set_dtype_requirements(self):
             ('range_50p_area',    np.float32, 'width, 50% area [ns]'),
             ('range_90p_area',    np.float32, 'width, 90% area [ns]'),
             ('rise_time',         np.float32, 'time between 10% and 50% area quantiles [ns]'),
-            ('area_fraction_top', np.float32, 'fraction of area seen by the top PMT array')
+            ('area_fraction_top', np.float32, 'fraction of area seen by the top PMT array'),
+            ('tight_coincidence', np.int16, 'Hits within tight range of mean'),
+            ('n_saturated_channels',      np.int16, 'Total number of saturated channels'),
+            ('tight_coincidence_channel', np.int16, 'PMT channel within tight range of mean'),
```

Comment:
every field we add here is multiplied by 4 in the total even-basics dtype (alt, main, S1, S2). Probably there is some story to as to why we need all these fields but we should carefully think about adding fields here because no-one ever removes fields from event_basics